### PR TITLE
Opto22 node red story

### PR DESCRIPTION
**Pull request by iskerrett**
New Opto22 customer story.  Still need to merge in graphic provided by Opto22.

### CODE REVIEW

File: src/customer-stories/opto22-embraces-node-red.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Opto 22 Embraces Node-RED to Drive Customer Innovation
+image: /images/stories/AdobeStock_362577635.jpeg
+date: 2023-06-01
+logo: /images/stories/logos/logo_test.jpg
+story:
+    brand: test
+    url:
```
Comment (joepavitt) [inline src/customer-stories/opto22-embraces-node-red.md:8]: I also suspect that this is meant to be a URL pointing to the Opto 22 website?

**Comment by Yndira-FlowForge** [comment]
@joepavitt I just added the image. Please, check it out, I don't think it looks as bad as I expected.
I looked for their website on google and added the URL. I also included a logo I found online. @iskerrett, please verify if those are the correct ones.
I didn't overlay the logo on the main stories page since it's already included in the image they provided.

**Comment by iskerrett** [comment]
This looks fine. I would like to suggest some changes to the logo on the full page.  

1. I don't think we need the Opto 22 name above the logo.
2. The logo should be on a transparent background. On a white background doesn't look great.

image

**Comment by Yndira-FlowForge** [comment]
> I don't think we need the Opto 22 name above the logo.

@iskerrett this has the link to their web, would you like me to remove it anyway?

**Comment by iskerrett** [comment]
> > I don't think we need the Opto 22 name above the logo.
> 
> @iskerrett this has the link to their web, would you like me to remove it anyway?

The logo also has a link to their website so I don't think we need it.  If we do, I would move it under the logo and show the full domain. Right now it looks like just the company name which I find confusing.

**Comment by joepavitt** [comment]
GitHub has gotten a little grumpy and isn't picking up latest changes to hte branch. Closing this PR to open a new one to see if that fixes it.

**Pull request closed without merging**